web/html/renew_idtoken.go

```diff
@@ -0,0 +1,108 @@
+package html
```

Comment:
please remove 'idtoken' from naming. We call it access token here. ID token is a little bit different thing. Call it just "renew_token"